internal/db/nosql/cassandra/option.go

```diff
@@ -125,23 +139,32 @@ func WithConnectTimeout(dur string) Option {
 	}
 }
 
+// WithPort returns the option to set the port number
 func WithPort(port int) Option {
 	return func(c *client) error {
-		c.port = port
+		if port > 0 {
```

Comment:
we have to check 0 or uint16 max

Reply:
should we return error if user set the port as negative value or >uint16 max?

Reply:
I think so, we should return error if invalid number passed